#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_esound_optional.dpatch by CJ van den Berg <cj@vdbonline.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: modify default.pa to make esound protocol optional

@DPATCH@

diff -Nru pulseaudio-0.9.5-orig/src/daemon/default.pa.in pulseaudio-0.9.5/src/daemon/default.pa.in
--- pulseaudio-0.9.5-orig/src/daemon/default.pa.in	2006-08-25 14:08:11.000000000 +0200
+++ pulseaudio-0.9.5/src/daemon/default.pa.in	2006-09-20 23:55:18.987673697 +0200
@@ -41,8 +41,7 @@
 #add-autoload-sink output module-alsa-sink sink_name=output
 #add-autoload-source input module-alsa-source source_name=input
 
-### Load several protocols
-load-module module-esound-protocol-unix
+### Load native protocol
 load-module module-native-protocol-unix
 
 ### Network access (may be configured with paprefs, so leave this commented
@@ -71,6 +70,9 @@
 
 .nofail
 
+### Load esound protocol
+load-module module-esound-protocol-unix
+
 ### Load something to the sample cache
 load-sample x11-bell /usr/share/sounds/gtk-events/activate.wav
 #load-sample-dir-lazy /usr/share/sounds/*.wav
